tips/templates/tips/style.css

```diff
@@ -0,0 +1,653 @@
+body{
```

Comment:
The file is very long, maybe you should split it?

Reply:
I don't think it is necessary